lib/models/BucketInfo.ts

```diff
@@ -37,6 +37,41 @@ export type VersioningConfiguration = {
     MfaDelete: any;
 };
 
+export type VeeamCapacity = {
```

Comment:
type should be either `VeamCapability` (to make it generic and support other veam capabilities in the feature) or `VeamSosApi` (and we would possibly have other VeamXXXX capabilities in the future)

Reply:
I chose `VeeamCapabilities` as it may contain more than SOSAPI, if needed: https://github.com/scality/Arsenal/pull/2059/files#diff-4ee3cd5de98073dadb72b27f034f4979480c04388cc4f4538f4895373527f174R40

Reply:
Ok, i'll use `VeeamSOSApi` for simplicity here, to avoid logic change in child PRs :smile: I'm fine with both anyway

Done & squashed in https://github.com/scality/Arsenal/pull/2059/commits/a6b234b7a8604d2c2d9dd49e2e35b1a7d9f8a50b#diff-4ee3cd5de98073dadb72b27f034f4979480c04388cc4f4538f4895373527f174R72